ncm-metaconfig/src/main/metaconfig/kibana/tests/regexps/config/base

```diff
@@ -6,5 +6,4 @@ Base test for config
 ^elasticsearch:$
 ^  hosts: http://mysupersecret.host.domain:9200$
 ^  ssl.verificationMode: none$
-^kibana:$
-^  index: .kibana$
+^  serviceAccountToken: token$
```

Comment:
The lines will be rendered in alphabetical order, so this line needs to be before line 8, i.e:
```
^  serviceAccountToken: token$
^  ssl.verificationMode: none$
```